extensions/vscode/src/eventErrors.ts

```diff
@@ -14,26 +14,40 @@ export function isCodedEventErrorMessage(
   return msg.errCode !== undefined;
 }
 
-type deploymentEvtErr = {
+type baseEvtErr = {
   dashboardUrl: string;
   localId: string;
   message: string;
   error: string;
 };
 
+type lockfileReadingEvtErr = baseEvtErr & {
+  lockfile: string;
+};
+
 export const isEvtErrDeploymentFailed = (
   emsg: EventStreamMessageErrorCoded,
-): emsg is EventStreamMessageErrorCoded<deploymentEvtErr> => {
+): emsg is EventStreamMessageErrorCoded<baseEvtErr> => {
   return emsg.errCode === "deployFailed";
 };
 
+export const isEvtErrRenvLockPackagesReadingFailed = (
+  emsg: EventStreamMessageErrorCoded,
+): emsg is EventStreamMessageErrorCoded<lockfileReadingEvtErr> => {
+  return emsg.errCode === "renvlockPackagesReadingError";
+};
+
 export const handleEventCodedError = (
   emsg: EventStreamMessageErrorCoded,
 ): string => {
   if (isEvtErrDeploymentFailed(emsg)) {
     return emsg.data.message;
   }
 
+  if (isEvtErrRenvLockPackagesReadingFailed(emsg)) {
+    return emsg.data.message;
```

Comment:
Minor nit: the error we show in the notification and the sidebar would be more user friendly if the path wasn't an absolute file path, but a relative path based on their workspace root.

It makes sense for the error in the deployment record to be an absolute file path since it doesn't know where the workspace is in VS Code (and it could change between work sessions).

To accomplish this we can do something like:

```suggestion
    return `Could not scan R packages from R package file: ${relativePath(Uri.parse(emsg.data.lockfile))}`;
```

This doesn't change the sidebar though. That would require a special case in `EvenEasierDeploy`.

Reply:
Ah, but it seems only the extension code knows the current VSCode workspace tree. I now better understand your point.